internal/applier/applier.go

```diff
@@ -0,0 +1,65 @@
+package applier
+
+import (
+	"context"
+
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	"github.com/timflannagan/platform-operators/api/v1alpha1"
+	"github.com/timflannagan/platform-operators/internal/sourcer"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	plainProvisionerID    = "core.rukpak.io/plain"
```

Comment:
Maybe we could pull these [constants](https://github.com/operator-framework/rukpak/blob/main/internal/provisioner/plain/types/types.go) in from Rukpak?

Reply:
The rukpak constants are defined in internal packages, so we'd either need to promote those packages to pkg/ or manually vendor them for now.